pkg/lib/clusterconfig/clusterconfig.go

```diff
@@ -21,42 +21,48 @@ import (
 
 	"github.com/cortexlabs/cortex/pkg/consts"
 	"github.com/cortexlabs/cortex/pkg/lib/aws"
+	"github.com/cortexlabs/cortex/pkg/lib/table"
+	kresource "k8s.io/apimachinery/pkg/api/resource"
+
 	cr "github.com/cortexlabs/cortex/pkg/lib/configreader"
 	"github.com/cortexlabs/cortex/pkg/lib/errors"
 	"github.com/cortexlabs/cortex/pkg/lib/hash"
+	"github.com/cortexlabs/cortex/pkg/lib/k8s"
 	"github.com/cortexlabs/cortex/pkg/lib/pointer"
 	"github.com/cortexlabs/cortex/pkg/lib/prompt"
-	"github.com/cortexlabs/cortex/pkg/lib/table"
 )
 
 type ClusterConfig struct {
-	InstanceType           *string `json:"instance_type" yaml:"instance_type"`
-	MinInstances           *int64  `json:"min_instances" yaml:"min_instances"`
-	MaxInstances           *int64  `json:"max_instances" yaml:"max_instances"`
-	ClusterName            string  `json:"cluster_name" yaml:"cluster_name"`
-	Region                 string  `json:"region" yaml:"region"`
-	Bucket                 string  `json:"bucket" yaml:"bucket"`
-	LogGroup               string  `json:"log_group" yaml:"log_group"`
-	Telemetry              bool    `json:"telemetry" yaml:"telemetry"`
-	ImagePredictorServe    string  `json:"image_predictor_serve" yaml:"image_predictor_serve"`
-	ImagePredictorServeGPU string  `json:"image_predictor_serve_gpu" yaml:"image_predictor_serve_gpu"`
-	ImageTFServe           string  `json:"image_tf_serve" yaml:"image_tf_serve"`
-	ImageTFServeGPU        string  `json:"image_tf_serve_gpu" yaml:"image_tf_serve_gpu"`
-	ImageTFAPI             string  `json:"image_tf_api" yaml:"image_tf_api"`
-	ImageONNXServe         string  `json:"image_onnx_serve" yaml:"image_onnx_serve"`
-	ImageONNXServeGPU      string  `json:"image_onnx_serve_gpu" yaml:"image_onnx_serve_gpu"`
-	ImageOperator          string  `json:"image_operator" yaml:"image_operator"`
-	ImageManager           string  `json:"image_manager" yaml:"image_manager"`
-	ImageDownloader        string  `json:"image_downloader" yaml:"image_downloader"`
-	ImageClusterAutoscaler string  `json:"image_cluster_autoscaler" yaml:"image_cluster_autoscaler"`
-	ImageMetricsServer     string  `json:"image_metrics_server" yaml:"image_metrics_server"`
-	ImageNvidia            string  `json:"image_nvidia" yaml:"image_nvidia"`
-	ImageFluentd           string  `json:"image_fluentd" yaml:"image_fluentd"`
-	ImageStatsd            string  `json:"image_statsd" yaml:"image_statsd"`
-	ImageIstioProxy        string  `json:"image_istio_proxy" yaml:"image_istio_proxy"`
-	ImageIstioPilot        string  `json:"image_istio_pilot" yaml:"image_istio_pilot"`
-	ImageIstioCitadel      string  `json:"image_istio_citadel" yaml:"image_istio_citadel"`
-	ImageIstioGalley       string  `json:"image_istio_galley" yaml:"image_istio_galley"`
+	InstanceType           *string       `json:"instance_type" yaml:"instance_type"`
+	InstanceCPU            *k8s.Quantity `json:"instance_cpu" yaml:"instance_cpu"`
```

Comment:
nit: since these three fields are not configured by the user, how do you feel about moving them to the bottom?